src/colony/modules/net.js

```diff
@@ -463,11 +473,11 @@ TCPSocket.prototype.__close = function (tryToClose) {
   function closeSocket(){
     if (self.socket === null) return;
     var ret = tm.tcp_close(self.socket);
-
     if (ret < 0 && ret != -tm.ENOTCONN) { // -57 is inactive, socket has already been closed
       if (retries > 3) {
         // tried 3 times and couldn't close, error out
-        self.emit('error', new Error('ENOENT Cannot close socket ' + self.socket + ' Got: err'+ret));
+        // removing this error for now, nothing for the user to do if this error occurs
```

Comment:
If this socket error occurs, what is going to happen downstream? Remove the commented out line and explain more what the reprecussion of not handling this error looks like.